test/cadet_web/admin_controllers/admin_assets_controller.ex

```diff
@@ -36,14 +36,14 @@ defmodule CadetWeb.AssetsControllerTest do
     @tag authenticate: :student
     test "GET /assets/:foldername", %{conn: conn} do
       conn = get(conn, build_url("testFolder"), %{})
-      assert response(conn, 403) =~ "User not allowed to manage assets"
+      assert response(conn, 403) =~ "Forbidden"
```

Comment:
We've lost the more informative error message, but I think there's a fair trade-off in terms of improving code de-duplication.